roofit/roofitcore/src/RooFormulaVar.cxx

```diff
@@ -173,7 +169,8 @@ void RooFormulaVar::printMultiline(ostream& os, Int_t contents, Bool_t verbose,
   if(verbose) {
     indent.Append("  ");
     os << indent;
-    formula().printMultiline(os,contents,verbose,indent);
+    if (_formula != nullptr)
+      _formula->printMultiline(os,contents,verbose,indent);
```

Comment:
before this patch, this would unconditionally call `printMultiline` and create `_formula` if needed, is the change in behavior intentional?